python/tests/integration/completions/test_chat_completion_with_function_calling.py

```diff
@@ -543,7 +543,7 @@ async def _test_helper(
             retries=5,
         )
 
-        if test_type != FunctionChoiceTestTypes.AUTO or stream:
+        if test_type != stream:
```

Comment:
Is this intended? `stream` is a bool while test_type is an enum.

Reply:
Good catch fixed!